cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -248,11 +242,10 @@ public BodyConsumer createAppVersion(HttpRequest request, HttpResponder responde
     if (Feature.LIFECYCLE_MANAGEMENT_EDIT.isEnabled(featureFlagsProvider)) {
       return create(request, responder, namespaceId, appId);
     }
-
-    ApplicationId applicationId = validateApplicationVersionId(namespaceId, appId, versionId);
+    validateApplicationId(namespaceId, appId);
```

Comment:
I don't think now we're ignoring the passed in version for this endpoint. When FF is off, we still allow creating custom versions. We're removing this API in 6.9 @sanjanasandeep correct me if I'm wrong

Reply:
yeah you are right, will change back